index.html

```diff
@@ -0,0 +1,190 @@
+        </div>
+        <h3 class="item-header">Exhibition</h3>
+        <p class="item-content">
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Fugit
+          libero sit in necessitatibus, ad sequi?
+        </p>
+      </article>
+      <article class="program-item flex">
+        <div class="item-icon">
+          <i class="fa fa-forumbee"></i>
+        </div>
+        <h3 class="item-header">Forum</h3>
+        <p class="item-content">
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Fugit
+          libero sit in necessitatibus, ad sequi?
+        </p>
+      </article>
+      <article class="program-item flex">
+        <div class="item-icon">
+          <i class="fa fa-building"></i>
+        </div>
+        <h3 class="item-header">Workshop</h3>
+        <p class="item-content">
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Fugit
+          libero sit in necessitatibus, ad sequi?
+        </p>
+      </article>
+      <article class="program-item flex">
+        <div class="item-icon">
+          <i class="fa fa-fire"></i>
+        </div>
+        <h3 class="item-header">Ignite</h3>
+        <p class="item-content">
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Fugit
+          libero sit in necessitatibus, ad sequi?
+        </p>
+      </article>
+    </div>
+    <a href="#main-program" class="join get-started top hidden-button">Join the Program</a>
+    <a href="#main-program" class="see-program hover-effect">SEE THE WHOLE PROGRAM</a>
+  </section>
+
+  <section class="featured-clubs flex" id="clubs"></section>
+
+  <section class="partners flex top" id="partners">
+    <h2 class="main-program-header top">Partners</h2>
+    <hr class="horizontal-line">
+    <div class="partner-items flex">
+      <div class="partner-icon">
+        <i class="fa fa-windows" aria-hidden="true"></i>
+      </div>
+      <div class="partner-icon">
+        <i class="fa fa-google" aria-hidden="true"></i>
+      </div>
+      <div class="partner-icon">
+        <i class="fa fa-apple" aria-hidden="true"></i>
+      </div>
+      <div class="partner-icon">
+        <i class="fa fa-amazon" aria-hidden="true"></i>
+      </div>
+      <div class="partner-icon">
+        <i class="fa fa-behance" aria-hidden="true"></i>
+      </div>
+    </div>
+  </section>
+
+  <footer class="footer flex top" id="contact">
+    <h3 class="featured-clubs-header">Contact</h3>
+    <hr class="horizontal-line">
+    <div class="flex contact-details">
+      <img src="images/small_logo.png" alt="university logo" class="ytu-logo">
+      <div class="adress">
+        <p class="phone-number"><b>Phone:</b> +123456789</p>
+        <p class="adress-details">
+          <b>Address:</b> Lorem, ipsum. <br>
+          Lorem ipsum dolor sit.<br>
+          34220 Lorem, ipsum dolor.
+        </p>
+      </div>
+    </div>
+    <p class="rights"><i class="fa fa-copyright"></i>2021. All rights reserved</p>
```

Comment:
- [ ]  As stated the project requirement The pages should look almost identical to the original design. So Kindly aligned the   "logo"  the `left` and "text" aligned to the `right` Just like the [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) 

The Design Guidelines Look likes this 

![Capture](https://user-images.githubusercontent.com/93598189/203607647-f2adee5b-e2fd-4d76-9298-210e7e7840a9.PNG)

Your Project Design Look likes this 

![Capture](https://user-images.githubusercontent.com/93598189/203607832-90b9e9a1-ad8b-43ae-bb39-d9d1054e3e13.PNG)